src/internal/common/dttm/dttm.go

```diff
@@ -98,10 +98,18 @@ func Now() string {
 	return FormatNow(Standard)
 }
 
+func OrNow(t time.Time) time.Time {
+	if (time.Time{}).Equal(t) {
```

Comment:
there's an `IsZero` function for time 😄

Reply:
oh, this is a duplicate due to the PR getting an update and me hitting refresh and my old comment not being shown

Reply:
done